internal/extsvc/azuredevops/client.go

```diff
@@ -141,5 +143,5 @@ type httpError struct {
 }
 
 func (e *httpError) Error() string {
-	return fmt.Sprintf("ADO API HTTP error: code=%d url=%q body=%q", e.StatusCode, e.URL, e.Body)
+	return fmt.Sprintf("AzureDevOps API HTTP error: code=%d url=%q body=%q", e.StatusCode, e.URL, e.Body)
```

Comment:
```suggestion
	return fmt.Sprintf("Azure DevOps API HTTP error: code=%d url=%q body=%q", e.StatusCode, e.URL, e.Body)
```

Nitpick, but I think it's `Azure DevOps`, two words